app/styles/partials/notifications.scss

```diff
@@ -162,3 +162,43 @@ $warning_color: #f44e06;
     opacity: 0;
   }
 }
+
+.notif-i{
+  padding: 20px 30px 34px 19px;
```

Comment:
Please do this without any of these styles. I don't think of these custom styles are required. Semantic-UI's list should be more than enough.